src/library/Uno.Themes.WinUI.Markup/Theme/CalendarView.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using Windows.UI;
+using Microsoft.UI.Text;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup;
+using Uno.Extensions.Markup.Internals;
+
+using ResourceKeyDefinitionAttribute = Uno.Themes.WinUI.Markup.ResourceKeyDefinitionAttribute;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static partial class CalendarView
+		{
+			public static partial class Resources
+			{
+				public static partial class Calendar
```

Comment:
Why is there an extra `Calendar` nesting here?

Reply:
this is an automatic grouping created for these properties:
- MaterialCalendarTodayForeground
- MaterialCalendarBlackoutForeground

the logic goes like this:
1. Material- prefix are discard due to special rule
2. -TodayForeground and -BlackoutForeground have the same name as properties on CalendarView, so they are considered "right-name"
2. this leaves "Calendar" for their "left-name"
3. since we have multiple elements with that same "left-name", a new group "Calendar" was created

(A missed point here is the removal of ControlType from the name, so instead of "-Calendar-", they were named "-CalendarView-" it would be trimmed down to "" and be promoted up one tier (Resources\Calendar\* -> Resources\*) for having an empty group name)

I will correct this, in the xaml, from the other PR.